test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Uses/dataset.toml

```diff
@@ -1,16 +1,16 @@
-data_source = "AEO2021 Reference Case"
+data_source = "NEMS Commercial Module"
 dataset_id = "aeo2021_ref_com_energy_end_uses"
 data_schema_type = "one_table"
-dataset_type = "benchmark" # <--- ?
-path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Uses/data/" # <---
+dataset_type = "benchmark"
+path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Uses/data/"
 description = "AEO 2021 Reference Case Commercial Energy End Uses Projection: 2013-2050"
-origin_creator = "Lixi Liu"
-origin_organization = "NREL"
+origin_creator = "Kevin Jarzomski"
+origin_organization = "EIA"
 origin_contributors = [ "Lixi Liu", "Meghan Mooney", "Elaine Hale",]
-origin_project = "Energy Benchmarking and Projection" # <---
+origin_project = "Dsgrid 2022 Commercial Buildings Projection"
```

Comment:
If these are the pure energy data directly from NEMS/AEO, I'd say the origin_project is AEO2021. I think "dsgrid 2022 Commercial Buildings Projection" is appropriate for the growth rates data.